### PR TITLE
Add data dictionary versioning

### PR DESCRIPTION
**Pull request by matentzn**
The resulting owl files get a new entry:

```
<owl:versionIRI rdf:resource="https://purl.ihccglobal.org/releases/2020-09-14/gcs.owl"/>
```

The TODAY variable can be set from the outside as well if so wanted.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -96,7 +97,7 @@ build/%.owl: build/intermediate/properties.owl templates/%.tsv build/intermediat
 	template \
 	--template $(word 3,$^) \
 	--merge-before \
-	annotate --ontology-iri "https://purl.ihccglobal.org/$(notdir $@)" \
+	annotate --ontology-iri "https://purl.ihccglobal.org/$(notdir $@)" --version-iri "https://purl.ihccglobal.org/releases/$(TODAY)/$(notdir $@)" \
```
Comment (jamesaoverton) [inline Makefile:100]: I would like to follow the OBO convention, which would make it:

```
--version-iri "https://purl.ihccglobal.org/$(notdir $(basename $@))/$(TODAY)/$(notdir $@)"
```

Right?

Comment (matentzn) [reply Makefile:100]: I thought of that, but won't this make it exceedingly hard to create a redirect? The suggestion I made had in mind a single redirect like this:

```
https://purl.ihccglobal.org/releases/ ---> https://github.com/IHCC-cohorts/data-harmonization/releases/download/v
```

If you would add the `$(notdir $(basename $@))` in the middle, you will have to create redirect rule for every single dictionary you create? Or is this something you will solve with some regex nginx hacking?

Comment (jamesaoverton) [reply Makefile:100]: The redirect rules won't be a problem.

Comment (jamesaoverton) [reply Makefile]: On second thought, please do

```
--version-iri "https://purl.ihccglobal.org/$(notdir $(basename $@))/releases/$(TODAY)/$(notdir $@)"
```

Comment (matentzn) [reply Makefile]: Done! :)

**Pull request merged**